src/util/passport.js

```diff
@@ -39,28 +39,42 @@ const localLogin = new LocalStrategy(localOptions, (email, password, done) => {
 
 const jwtOptions = {
   jwtFromRequest: ExtractJwt.fromHeader('authorization'),
-  secretOrKey: process.env.JWT_SECRET
+  secretOrKey: process.env.JWT_SECRET,
+  passReqToCallback: true
 };
 
-const jwtLogin = new Strategy(jwtOptions, (payload, done) => {
-  return knex('role').select('email').where('id', payload.sub)
-  .then(rows => {
-    if (_.isEmpty(rows)) {
-      return done(null, false)
-    }
-    const startTime = moment(payload.iat)
-    const nowTime = moment(new Date().getTime())
-    const timeSpent = nowTime.diff(startTime, 'hours')
-    if (timeSpent > 24) {
-      return done(null, false)
-    } else {
-      const id = payload.sub;
-      return done(null, id)
-    }
-  });
+const jwtLogin = new Strategy(jwtOptions, (req, payload, done) => {
+  const uuid = req.headers['x-user-uuid'];
+  return knex('users').select('id').where('uuid', uuid)
+    .then(rows => {
+      if (_.isEmpty(rows)) {
+        return knex('role').select('email').where('id', payload.sub)
```

Comment:
There's multiple database queries in a row. I'd consider using `transactions`.

Reply:
Will look into this.